.github/workflows/ci-build.yml

```diff
@@ -71,6 +71,19 @@ jobs:
           command: fmt
           args: --all -- --check
 
+      # due to sqlx dependency, prepare environment before building this package
+      - name: Pre-build steps for `node-status-api`
+        run: |
+          cargo install sqlx-cli
```

Comment:
heads up, let's add this to the runner image as a follow up to this PR so we don't have to build it on each run